docs/reference/modbus.md

```diff
@@ -133,4 +133,17 @@ timeout: 2s # timeout, without unit in ns
 
 Bei den `int32s/uint32s` Dekodierungen wird die Wortreihenfolge vertauscht und sind z.B. bei E3/DC Geräten nützlich.
 
-Um ein Regsiter zu schreiben wird `type: writesingle` verwendet, welches ein einzelnes 16bit Register (entweder `int` oder `bool`) schreibt. Die Kodierung ist hier immer `uint16`.
+### Schreiben von Registern
+Es können sowohl Holding-Register als auch Coils beschrieben werden. Dazu muss entweder `type: writeholding` für Holding-Register oder `type: writecoil` für Coils angegeben werden.
+`type: writeholding` schreibt immer ein 16Bit Register (int oder bool16). Die Kodierung ist hier immer `uint16`.
```

Comment:
> Die Kodierung ist hier immer `uint16`.

Hat das für den Anwender eine Auswirkung?

Reply:
Naja, das habe ich nur so von der usrsprpünglichen Doku übernommen. 
So wie ich den Code verstehe, muss man zwar was angeben, es sopielt aber keine Rolle, was, da immer ein single Register geschrieben wird. HAbe daher in der Doku behauptet, man müsse immer `uint16 ` angeben.
Besser wäre es, analog zu `writecoil `intern immer `uint16 ` zu setzen. Wollte den code aber jetzt nicht ändern.